docs/content/dagster-cloud/managing-deployments/dagster-cloud-yaml.mdx

```diff
@@ -0,0 +1,143 @@
+  - location_name: machine_learning
+    code_source:
+      python_file: ml/ml_model.py
+```
+
+## Configuring your working directory
+
+There might be some cases where you want to run your Dagster code from a different directory than the root of your code repository. You can specify the directory you want to run your Dagster code from in the dagster_cloud.yaml file using the `working_directory` option.
+
+Parameter reference table:
+
+| Parameter           | Description                                            | Data type     |
+| ------------------- | ------------------------------------------------------ | ------------- |
+| `working_directory` | The path of the directory that Dagster should run from | string (path) |
+
+```yaml
+#dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+    working_directory: ./project_directory
+```
+
+## Build directory
+
+Setting a build directory is useful if your setup.py or requirements.txt is not at your project root but in a subdirectory. This is common if you have multiple python packages within a single dagster project.
+
+The build registry is applicable to hybrid deployments only.
+
+Parameter reference table:
+
+| Parameter                   | Description                                                                                                                                                       | Data type                |
+| --------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------ |
+| `locations.build.directory` | The directory in your project code that you want to deploy, the default is `.`. If you have sub-folders you can specific the path to only deploy that sub-project | string (path)            |
+| `locations.build.registry`  | The Docker registry to push your code location to in the registry (Hybrid only)                                                                                   | string (docker registry) |
+
+Example:
+
+```yaml
+#dagster_cloud.yaml
+
+locations:
+  - location_name: data-eng-pipeline
+    code_source:
+      package_name: quickstart_etl
+    build:
+      directory: ./
+      registry: your-docker-image-registry/image-name
+```
+
+## Python Executable
+
+If you are using Dagster Cloud Serverless, the Python executable to use is specified [here](https://docs.dagster.io/dagster-cloud/deployment/serverless#using-a-different-python-version). For Dagster Cloud Hybrid, the python executable that is installed globally in the image, or the default Python executable on the local system if you use the local agent, will be used. If you want to use a different python executable, you can specify it in the dagster_cloud.yaml file. It might also be useful to have different python executables for different code locations.
```

Comment:
I think we can rework this paragraph a bit to make the important parts stand out - for example:

> Depending on the [deployment type](/dagster-cloud/deployment) you're using, the `executable_path` parameter does different things:
>
> - **Serverless** - [copy - see my second comment about this]
> - **Hybrid** - If using a [local agent](/dagster-cloud/deployment/agents/local), this specifies the default Python executable to use on the local system. If using other [agent types](/dagster-cloud/deployment/agents), this specifies the Python executable installed globally in the image.
>  If using multiple code locations, you can specify a different Python executable for each location.

Reply:
For the Serverless copy - I'm not sure what I'm supposed to take away from what the link points to. What should I change this value to, if anything?

Reply:
updated so don't think this comment is relevant anymore but will leave it in for now